### ISSUE TITLE
mixup_labels

### ISSUE THREAD

**Issue by HanAccount** [mancinimassimiliano/CuMix#8]
Hi! Why do synthesized labels(mixup_labels) have decimals?
I wanted to use it as a data enhancement to add to the method, but found that mixup_labels is not an integer.

When I put training set labels and mixup_labels cat after cross entropy float type error
How do I turn mixup_labels into integers?

thanks!
![image](https://user-images.githubusercontent.com/87269265/172294973-be60466b-198f-4f58-9770-a46de73407dd.png)
